nodejs/json-rpc-relay/integration.tests/get.balance.test.js

```diff
@@ -0,0 +1,67 @@
+console.clear();
+const axios = require(`axios`);
+require('dotenv').config({path: '.env'});
+const Web3 = require('web3');
+const { ethers } = require("ethers");
+
+const arkhiaJsonRpcRelayMainnet = `${process.env.ARKHIA_MAINNET_URL}/${process.env.ARKHIA_API_KEY}`;
+const arkhiaJsonRpcRelayTestnet  = `${process.env.ARKHIA_TESTNET_URL}/${process.env.ARKHIA_API_KEY}`;
+const communityHashioMainnet = process.env.COMMUNITY_MAINNET_URL;
+const testnetEvmAccount = "0x0000000000000000000000000000000000000589";
+
+const getAccountPayload = () => {
+    const data = JSON.stringify({
+        "jsonrpc": "2.0",
+        "method": "eth_getBalance",
+        "params": [
+            "0x0000000000000000000000000000000000000589"
+        ],
+        "id": 1
+      });
+    return data;
+}
+
+describe('[CURL] Accounts',  () => {
+
+    // WIP
+    test('Should get Balance from Hedera account through EVM Account address from Arhia Testnet', async () => {
```

Comment:
Not getting this in, will come back to it

Reply:
No worries